string name
uint32 num_points
flyer_controller/AutosequencePoint[] points